src/utils/coordinates/coordinateUtils.js

```diff
@@ -11,13 +14,13 @@ import { LV03, LV95, WEBMERCATOR, WGS84 } from './coordinateSystems'
  *
  * @param x {Number} X part of the coordinate (Easting/Longitude)
  * @param y {Number} Y part of the coordinate (Northing/Latitude)
- * @returns {Number[]} Re-projected coordinate as [lat, lon], or undefined if the input was not
+ * @returns {Number[]} Re-projected coordinate as [lon, lat], or undefined if the input was not
  *   convertible to WebMercator
  */
 export const reprojectUnknownSrsCoordsToWGS84 = (x, y) => {
-    if (LV95.getBoundsAs(WEBMERCATOR)?.isInBounds(x, y)) {
+    // guessing if this is already WGS84, or Mercator or a Swiss projection (if so, we need to reproject it to WGS84)
+    if (LV95BoundsAsWebMercator.isInBounds(x, y)) {
```

Comment:
In this function why checking backward ? I think it would make our life easier if we only support normal axis order and not try to guess wrong axis order.

Reply:
That's because of the issue with `CRS:84` and `WGS:84`, some maps function with a lat/lon pairing, and others with lon/lat, and you can't be sure it will be either one of them you receive.

For LV95 and LV03 backward checking, it is to mimic what was done in `mf-geoadmin3` and help automatically correct users' mistake with X/Y